openmc/filter.py

```diff
@@ -1875,6 +1875,8 @@ class EnergyFunctionFilter(Filter):
         A grid of energy values in [eV]
     y : iterable of Real
         A grid of interpolant values in [eV]
+    interpolation : str
+        The type of interpolation to be used.
```

Comment:
Is it worth putting in the options ```Interpolation scheme: {'histogram', 'linear-linear', 'linear-log',
        'log-linear', 'log-log', 'quadratic', 'cubic'}``` here as well

Reply:
Good call. Useful to have it there as well. Thanks!